server/src/app.js

```diff
@@ -5,17 +5,15 @@ import cors from 'cors';
 import chalk from 'chalk';
 import { db } from './models/index';
 import dbConn from './startup/dbConnection';
+import router from './routes/index';
 
 const app = express();
 const port = process.env.PORT || 5000;
 
 app.use(cors());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-
-app.get('/', (req, res) => {
-  res.send('Welcome to the backend of Library');
-});
+app.use('/api/v1', router);
 
```

Comment:
I moved it inside routes/index since that is where we are calling all the routes and I created a one liner code for the version control as you can see just below the removed code. It takes care of all the routes

Reply:
`app.get('/', (req, res) => {
  res.send('Welcome to the backend of Library');
});`
I actually meant this line of code?